jme3-core/src/main/java/com/jme3/export/SavableClassUtil.java

```diff
@@ -208,4 +223,27 @@ public static Savable fromName(String className, List<ClassLoader> loaders) thro
 
         return fromName(className);
     }
+
+    /**
+     * Use reflection to gain access to the no-arg constructor of the named
+     * class.
+     *
+     * @return the pre-existing constructor, or null if the class lacks a no-arg
+     * constructor
+     */
+    private static Constructor findNoArgConstructor(String className)
+            throws ClassNotFoundException {
+        Class clazz = Class.forName(className);
+        Constructor[] allConstructors = clazz.getDeclaredConstructors();
```

Comment:
You could use `getDeclaredConstructor()` this instead of getting all constructors `getDeclaredConstructors()`

```
Constructor<?> constructor;
  try {
    constructor = type.getDeclaredConstructor();
  } catch (NoSuchMethodException e) {
    throw new Exception("Please ensure that the type '" + type.getSimpleName() + "' has a no-arg constructor.");
  }
  return constructor;
```

Reply:
OK, I'll use `getDeclaredConstructor()` instead of the for-loop.